### DISCUSSION TITLE
Running watchtower container via cron

### DISCUSSION THREAD

**Discussion by milindpatel63** [containrrr/watchtower#1106] [Q&A] [answered]
Is it possible to run the watchtower container using the command "docker start watchtower" via a cron schedule in a linux distro..?
I'll have to set the restart policy to "no" for that....
Will watchtower update itself if i do it this way?

This solves the following issues for me:
Healthchecks integration - i can basically run healthchecks ping command together with docker start command...so no healthchecks integration inside watchtower itself is needed......or i can even use https://github.com/bdd/runitor to capture and send watchtower log to healthchecks...

**Comment by simonaronsson** [comment, marked as answer]
Yes. In this case, you should probably also use `--run-once`.
